investor_wallet_platform_base/views/invoice_view.xml

```diff
@@ -27,8 +27,8 @@
         <field name="inherit_id" ref="account.view_account_payment_invoice_form"/>
         <field name="arch" type="xml">
             <field name="journal_id" position="attributes">
-                <attribute name="invisible">True</attribute>
-            </field>
+				<attribute name="invisible">True</attribute>
+	        </field>
```

Comment:
thanks for having brought back attribute line. but indentation seems weird.

Reply:
damn